src/inputs.ts

```diff
@@ -39,13 +39,13 @@ export async function promptForFavoriteColorName(color: string) {
   return inputName || '';
 }
 
-export async function promptForFavoriteColor() {
+export async function promptForFavoriteColor(isPrimaryEnvironment: boolean = false) {
```

Comment:
@johnpapa I have to pass isPrimaryEnvironement to this code path so that the memento is not overwritten in changeColor()